src/main/java/com/sample/activities/DropDownMethods.java

```diff
@@ -0,0 +1,30 @@
+package com.sample.activities;
+
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class DropDownMethods extends FindMethods{
+	
+	/*
+	 * This method locate Select drop down elements and then select user input from paramaeter list.
+	 * Ataul
+	 * 8-16-2016 13:55
+	 */		
+			
+	public void selectFromDropDownList(String locator,String attributeOfLocator,String value){
+		
+		try{			
+			WebElement element = getElement(locator, attributeOfLocator);
+			Select  dvalue = new Select(element);
+			dvalue.selectByVisibleText(value);
+		} catch(NoSuchElementException e){
+			
```

Comment:
There will be no System Out Statement. Write Asser.file to get result in Allure report